scripts/supervisord/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#
+# This is an "initContainer" image using the base "source-to-image" OpenShift template
+# in order to apprpriately inject the supervisord binary into the application container.
+#
+# We use the base s2i image in order to use binaries such as "fix-permissions" to
+# add approriate file/folder permissions to nodejs / ruby / python applications.
+# 
+
+FROM centos/s2i-base-centos7
```

Comment:
why are we using s2i base image? We don't need any s2i capability in this image, or do we?
We should probably use something really tiny something like busybox, as the time it takes to create new component will mostly depend on the time it takes to pull this image.

Reply:
This is no needed and could start from busybox as we did here -> https://goo.gl/fbV7GS

Reply:
@kadel I included using the s2i base image as it has the `fix-permissions` variable.

You are right however, we can use `busybox` or `centos` instead.